NEWS.md

```diff
@@ -1,3 +1,26 @@
+## 11.5.0-SNAPSHOT 2019-05-24
```

Comment:
The [release procedures](https://dev.folio.org/guidelines/release-procedures/#prepare-the-news-document) state that the NEWS should not be updated during pull requests, and only on formal releases.

I wasn't aware that endpoint documentation was intended to be included in releases notes.

@skoczko Are we expected to publish endpoint documentation as part of release notes?

Reply:
@marcjohnson-kint you are right, I will revert my changes for news.md